server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/Serializer.scala

```diff
@@ -216,11 +219,84 @@ class Serializer @Inject() (mailboxIdFactory: MailboxId.Factory) {
   private implicit val propertiesRead: Reads[Properties] = Json.valueReads[Properties]
   private implicit val mailboxGetRequest: Reads[MailboxGetRequest] = Json.reads[MailboxGetRequest]
 
+
+  private implicit val mailboxRemoveEmailsOnDestroy: Reads[RemoveEmailsOnDestroy] = Json.reads[RemoveEmailsOnDestroy]
```

Comment:
Shouldn't that be an other ticket? (related to /set delete? #3516  )

Reply:
Yes it will be filled in another ticket. But as we followed the RFC to create the Request and Response object. I see no harm in having the skeleton of those objects right now.